mappings/net/minecraft/util/math/weight/WeightedList.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_6012 net/minecraft/util/math/weight/WeightedList
```

Comment:
There is already a class named WeightedList in nm/util/collection. What's the difference(s)?